node-installer/internal/config/config.go

```diff
@@ -25,19 +22,6 @@ type Config struct {
 
 // Validate validates the configuration.
 func (c Config) Validate() error {
-	if c.RuntimeHandlerName == "" {
-		return errors.New("runtimeHandlerName is required")
-	}
-	if len(c.RuntimeHandlerName) > 63 {
-		return errors.New("runtimeHandlerName must be 63 characters or fewer")
-	}
-	matched, err := regexp.Match(`^[a-z0-9]([-a-z0-9]*[a-z0-9])?$`, []byte(c.RuntimeHandlerName))
-	if err != nil {
-		return err
-	}
-	if !matched {
-		return errors.New("runtimeHandlerName must be a lowercase RFC 1123 subdomain")
-	}
```

Comment:
Release tests makes sense. I'm not even sure how much sense it'd make to put this into the node-installer, since it's deployed with the actual `runtimeClass` Kubernetes resource, which should [fail to deploy](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names) in that case?